src/sentry/integrations/discord/requests/base.py

```diff
@@ -116,6 +116,16 @@ def authorize(self) -> None:
         timestamp: str | None = self.request.META.get("HTTP_X_SIGNATURE_TIMESTAMP")
         body: str = self.request.body.decode("utf-8")
 
+        logger.info(
+            "discord.authorize.request",
+            extra={
+                "public_key": public_key,
+                "signature": signature,
+                "timestamp": timestamp,
+                "body": body,
+            },
+        )
+
         if signature and timestamp and verify_signature(public_key, signature, timestamp + body):
```

Comment:
@17hogeju let's add a logging message if it fails so we know that it's from this code block